x/nodes/types/msg_test.go

```diff
@@ -663,6 +664,47 @@ func TestMsgStake_ValidateBasic(t *testing.T) {
 	}
 }
 
+func TestMsgStake_Delegators(t *testing.T) {
```

Comment:
Wdyt of having an upper bound on the number of delegators? I think you can imagine the on-chain bloat attacks this could cause.

Reply:
Good point.  Instead of directly limiting the number of delegators, I decided to disallow zero share delegators.  With that, the number of delegators is limited to 100 because the total share cannot exceed 100.